Program.cs

```diff
@@ -1,14 +1,9 @@
 ﻿// Check whether the given number is a Prime Number or not
 
 Console.Write ("Enter a number: ");
-string input = Console.ReadLine ();
-int given_number = int.Parse (input);
-if (given_number < 0) {
-   Console.WriteLine ("Negative Numbers cannot be prime.");
-}
-if (given_number is 0 or 1) {
-   Console.WriteLine ("{0} is neither prime nor composite.", given_number);
-}
+int given_number = int.Parse (Console.ReadLine ());

```

Comment:
No underscore in variable name. Fix this error
![image](https://github.com/RaaneshKumar/Training/assets/94427955/83113ed6-d7a6-4572-a743-3c621a3b6ef3)